LSP-css.sublime-settings

```diff
@@ -7,21 +7,115 @@
 				"scopes": ["source.css"],
 				"syntaxes": [
 					"Packages/CSS/CSS.sublime-syntax",
-					"Packages/CSS3/CSS3.sublime-syntax"
-				]
+					"Packages/CSS3/CSS3.sublime-syntax",
+				],
 			},
 			{
 				"languageId": "scss",
 				"scopes": ["source.scss"],
-				"syntaxes": ["Packages/Sass/Syntaxes/SCSS.sublime-syntax"]
+				"syntaxes": ["Packages/Sass/Syntaxes/SCSS.sublime-syntax"],
 			},
 			{
 				"languageId": "less",
 				"scopes": ["source.less"],
-				"syntaxes": ["Packages/LESS/Syntaxes/LESS.sublime-syntax"]
-			}
+				"syntaxes": ["Packages/LESS/Syntaxes/LESS.sublime-syntax"],
+			},
 		],
-		"initializationOptions": {},
-		"settings": {}
-	}
-}
\ No newline at end of file
+		"initializationOptions": {
+			"dataPaths": [],
+		},
+		"settings": {
+			"css": {
+				"customData": [],
+				"completion": {
+					"triggerPropertyValueCompletion": true,
+					"completePropertyWithSemicolon": true,
+				},
+				"validate": true,
+				"lint": {
+					"compatibleVendorPrefixes": "ignore",
+					"vendorPrefix": "warning",
+					"duplicateProperties": "ignore",
+					"emptyRules": "warning",
+					"importStatement": "ignore",
+					"boxModel": "ignore",
+					"universalSelector": "ignore",
+					"zeroUnits": "ignore",
+					"fontFaceProperties": "warning",
+					"hexColorLength": "error",
+					"argumentsInColorFunction": "error",
+					"unknownProperties": "warning",
+					"validProperties": [],
+					"ieHack": "ignore",
+					"unknownVendorSpecificProperties": "ignore",
+					"propertyIgnoredDueToDisplay": "warning",
+					"important": "ignore",
+					"float": "ignore",
+					"idSelector": "ignore",
+					"unknownAtRules": "warning",
+				},
+				"trace": {
```

Comment:
This is something I've toggled to trace communication. We shouldn't add it to default settings.

Reply:
I mean the trace option.
(I'm on mobile and reviewing is horrible here :))

Reply:
no wonder there is only that one is on :laughing: